moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -618,5 +618,5 @@ class RobotModel
 }  // namespace core
 }  // namespace moveit
 
-namespace robot_model = moveit::core;
-namespace robot_state = moveit::core;
+namespace[[deprecated("use moveit::core")]] robot_model = moveit::core;
```

Comment:
This syntax breaks Travis. Don't have the time to lookup the correct one, though.